src/pages/Error/_shared.tsx

```diff
@@ -13,7 +13,7 @@ export function LinkContactSupport(): JSX.Element {
 
 export function LinkVisitHomepage(): JSX.Element {
   return (
-    <Link href='/' className='a-btn a-btn__full-on-xs visit-homepage'>
+    <Link href='/landing' className='a-btn a-btn__full-on-xs visit-homepage'>
```

Comment:
I think this one also needs to stay `/`, since we don't know if they're logged in or not. Unlike my other comments, this one probably needs to stay `/` even if we have conditional routing about if they're logged in or not and where they're coming from, since we want to send them to a hopefully known "good" state which more than likely will be the home page.